packages/shared/src/components/post/write/SquadsDropdown.tsx

```diff
@@ -44,13 +54,18 @@ export function SquadsDropdown({
         menu: 'menu-secondary',
         item: 'h-auto',
       }}
+      shouldIndicateSelected={false}
       selectedIndex={selected}
       onChange={(_, index) => onSelect(index)}
       options={squadsList}
       scrollable
       data-testid="timezone_dropdown"
       renderItem={renderDropdownItem}
-      drawerProps={{ isFullScreen: true }}
+      drawerProps={{
+        isFullScreen: true,
+        title: 'Choose a Squad',
+        className: { drawer: 'p-0 pr-2', title: 'mb-4' },
```

Comment:
Do you need the p-0 even then?
If the existing one has padding shouldn't it be !p-0?

Reply:
The classes on the base drawer component have it like `className ?? 'px-2 py-3`. So passing any class would have it removed hence not needing the `!`.